.travis.yml

```diff
@@ -2,32 +2,32 @@
 sudo: false
 language: python
 env:
-  - TOXENV=py27-django18-sqlite
-  - TOXENV=py27-django18-mysql
-  - TOXENV=py27-django18-postgres
+  - TOXENV=py27-django18-sqlite PYTHONPATH=$HOME/virtualenv/python2.7.9/lib/python2.7/site-packages  TOX_TESTENV_PASSENV="PYTHONPATH"
```

Comment:
Thanks for explanation, @phlax! As said, mind adding a comment in the file so everyone reading it is aware of it? TIA.